pkg/devfile/validate/commands.go

```diff
@@ -1,115 +1,52 @@
 package validate
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/openshift/odo/pkg/devfile/parser/data/common"
+	genericValidation "github.com/openshift/odo/pkg/devfile/validate/generic"
```

Comment:
I think the alias also not needed, generic.ValidateCommands is pretty intuitive

Reply:
updated!, pls help review! 

I've also added a single point of validation call for generic validation like you suggested and it makes sense.